app/src/main/java/ch/hikemate/app/ui/map/MapScreen.kt

```diff
@@ -180,6 +180,7 @@ fun showHikeOnMap(mapView: MapView, hike: HikeRoute, color: Int) {
  */
 fun clearHikesFromMap(mapView: MapView) {
   mapView.overlays.clear()
+  mapView.invalidate()
```

Comment:
Question: Is invalidate necessary? Does clear not already handle everything?

Reply:
This indeed fixes a little subtle bug where, when you start the app without moving the map and search for hikes, hikes were not appearing until you move the map